home/menu/DevMenuTaskInfo.tsx

```diff
@@ -1,45 +1,116 @@
+}
+
 export function DevMenuTaskInfo({ task }: Props) {
-  const manifest = task.manifestString && JSON.parse(task.manifestString);
-  const iconUrl = manifest && (manifest.iconUrl ?? manifest.extra?.expoClient?.iconUrl);
-  const taskName = manifest && (manifest.name ?? manifest.extra?.expoClient?.name);
-  const sdkVersion = manifest && (manifest.sdkVersion ?? manifest.extra?.expoClient?.sdkVersion);
-  const runtimeVersion = manifest && manifest.runtimeVersion;
+  const theme = useExpoTheme();
+
+  const manifest = task.manifestString
+    ? (JSON.parse(task.manifestString) as typeof Constants.manifest | typeof Constants.manifest2)
+    : null;
+  const manifestInfo = manifest ? getInfoFromManifest(manifest) : null;
 
   return (
     <View>
       <Row bg="default" padding="medium">
-        {!manifest?.metadata?.branchName && iconUrl ? (
-          // EAS Updates don't have icons
-          <Image source={{ uri: iconUrl }} style={styles.taskIcon} />
+        {manifestInfo?.iconUrl ? (
+          <Image source={{ uri: manifestInfo.iconUrl }} style={styles.taskIcon} />
         ) : null}
         <View flex="1" style={{ justifyContent: 'center' }}>
           <Text type="InterBold" color="default" size="medium" numberOfLines={1}>
-            {taskName ? taskName : 'Untitled Experience'}
+            {manifestInfo?.taskName ? manifestInfo.taskName : 'Untitled Experience'}
           </Text>
-          {sdkVersion && (
+          {manifestInfo?.sdkVersion && (
             <Text size="small" type="InterRegular" color="secondary">
               SDK version:{' '}
               <Text type="InterSemiBold" color="secondary" size="small">
-                {sdkVersion}
+                {manifestInfo.sdkVersion}
               </Text>
             </Text>
           )}
-          {runtimeVersion && (
+          {manifestInfo?.runtimeVersion && (
             <Text size="small" type="InterRegular" color="secondary">
               Runtime version:{' '}
               <Text type="InterSemiBold" color="secondary" size="small">
-                {runtimeVersion}
+                {manifestInfo.runtimeVersion}
               </Text>
             </Text>
           )}
+          {!manifestInfo?.isVerified && (
+            <Row
+              bg="warning"
+              border="warning"
+              rounded="medium"
+              padding="0"
+              align="center"
+              style={{
+                alignSelf: 'flex-start',
+                marginTop: 3,
+              }}>
+              <Ionicons
+                name={Platform.select({ ios: 'ios-warning', default: 'md-warning' })}
+                size={14}
+                color={theme.text.warning}
+                lightColor={theme.text.warning}
+                darkColor={theme.text.warning}
+                style={{
+                  marginHorizontal: 4,
+                }}
+              />
+              <Text
+                color="warning"
+                type="InterRegular"
```

Comment:
Might be nice to increase the weight of this text

```suggestion
                type="InterSemiBold"
```